vue/src/views/HomeView.vue

```diff
@@ -1,18 +1,181 @@
+            <el-select v-model="data.over_or_less">
+              <el-option label="以上" value="over" />
+              <el-option label="以下" value="less" />
+            </el-select>
+          </el-col>
+        </el-row>
+        <p :class="$style.right">になったら通知する</p>
+        <el-row>
+          <el-col :span="4">
+            <el-icon :size="30"><Memo /></el-icon><br />
+            <small>メモ</small><br />
+          </el-col>
+          <el-col :span="16">
+            <el-input type="textarea" v-model="data.about" />
+          </el-col>
+          <el-col :span="4" @click="save(key)">
+            <el-icon :size="30"><UploadFilled /></el-icon><br />
+          </el-col>
+        </el-row>
+      </el-card>
+    </div>
+
+    <div :id="$style.footer">
+      <el-button @click="location.href = 'https://ambidata.io/bd/board.html?id=55252'">
+        家事の発生状況を見る
+        <el-icon :size="30"><TrendCharts /></el-icon>
+      </el-button>
+      <el-select :id="$style.lang" v-model="state.lang">
+        <el-option @click="lagnChange" label="JP" value="jp" />
+        <el-option @click="lagnChange" label="EK" value="ek" />
+      </el-select>
+    </div>
   </div>
 </template>
 
 <script lang="ts">
-import { defineComponent } from 'vue';
-import HelloWorld from '@/components/HelloWorld.vue'; // @ is an alias to /src
+import { defineComponent, reactive } from 'vue';
+import { TrendCharts, UploadFilled, Memo } from '@element-plus/icons-vue';
+import * as notification from '@/components/notification';
 
 export default defineComponent({
   name: 'HomeView',
   components: {
-    HelloWorld,
+    TrendCharts,
+    UploadFilled,
+    Memo,
+  },
+  setup() {
+    const state = reactive({
+      lang: 'jp',
+      sensor: {
+        d1: {
+          active: true,
+          about: '選択済みタオル枚数が減ったとき(箱の天井との距離が220以上のとき)',
+          threshold: 220,
+          over_or_less: 'less',
+        },
+        d2: {
+          active: true,
+          about: '選択済みタオル枚数が減ったとき(タオル重量が360以下のとき)',
+          threshold: 360,
+          over_or_less: 'less',
+        },
+        d3: {
+          active: false,
+          about: '',
+          threshold: 0,
+          over_or_less: '',
+        },
+        d4: {
+          active: false,
+          about: '',
+          threshold: 0,
+          over_or_less: '',
+        },
+        d5: {
+          active: false,
+          about: '',
+          threshold: 0,
+          over_or_less: '',
+        },
+        d6: {
+          active: false,
+          about: '',
+          threshold: 0,
+          over_or_less: '',
+        },
+        d7: {
+          active: false,
+          about: '',
+          threshold: 0,
+          over_or_less: '',
+        },
+        d8: {
+          active: false,
+          about: '',
+          threshold: 0,
+          over_or_less: '',
+        },
+      },
+    });
+    const save = (key: string) => {
+      notification.success({ title: '成功', message: `${key}を保存しました` });
```

Comment:
todo: d1, d2 とかだと分かりづらいので保存だけか、第一、第二みたいな表現にする
```suggestion
      notification.success({ title: '成功', message: '保存しました' });
```